qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -172,15 +189,20 @@ def clifford_2_qubit_circuit(self, num):
         if form == 3:
             qc.cx(0, 1)
         if form in (1, 2):
-            if k0 == 1:
-                qc._append(VGate(), [qr[0]], [])
-            if k0 == 2:
-                qc._append(WGate(), [qr[0]], [])
-            if k1 == 1:
-                qc._append(VGate(), [qr[1]], [])
-            if k1 == 2:
-                qc._append(VGate(), [qr[1]], [])
-                qc._append(VGate(), [qr[1]], [])
+            if k0 == 1:  # V gate
+                qc.sdg(0)
+                qc.h(0)
+            if k0 == 2:  # W gate
+                qc.h(0)
+                qc.s(0)
+            if k1 == 1:  # V gate
+                qc.sdg(1)
+                qc.h(1)
+            if k1 == 2:  # W gate (replacing these with qc.h(1) + qc.s(1) causes a test failure)
```

Comment:
Any idea which test fails and why? there should be no difference between the case where k0=2 and k1=2 (both should append a W=HS gate)

Reply:
This test: https://github.com/Qiskit/qiskit-experiments/blob/622554d475f1b0ee7c20941e1ad79613a51771e7/test/library/randomized_benchmarking/test_randomized_benchmarking.py#L579
(See also https://github.com/Qiskit/qiskit-experiments/runs/8031956851?check_suite_focus=true for the error message)
Honestly, I have no idea... I'd like to ask it to the person who write the original code. I tried to replace the W with HS and noticed this wired behavior. And for safety, I just leave it as is...

Reply:
I think that it's better to fix the test, since the code should definitely be correct (maybe it's only a matter of error bounds?)

Reply:
I've updated the test following @nkanazawa1989 's suggestion in [1972130](https://github.com/Qiskit/qiskit-experiments/pull/898/commits/1972130e8c4ded6c1416b5b2ba9942c2714a367e).